BasementRenovator.py

```diff
@@ -3601,7 +3638,7 @@ def data(self, index, role=Qt.DisplayRole):
 
         elif role == Qt.DisplayRole:
             if isinstance(item, EntityGroupItem):
-                return item.name
+                return item.name + (" >" if item.collapsed else "")
```

Comment:
Let's use this character to better visually separate from the ASCII: ▶ (note this is supposed to be a *black* right facing triangle. This emoji representation is inaccurate to what BR (hopefully) will render)